test/driver_sanity/kernel_compat/scripts/vagrant_scap_open_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [[ $# -ne 1 || "${EUID}" -ne 0 ]]; then
+  echo "Usage: sudo bash vagrant_scap_open_test.sh COMPILER"
+  echo "Run as root user in VM"
+  exit 1
+fi
+
+COMPILER="${1}";
+set -eou pipefail
+
+echo "Test run for compiler/kernel ${COMPILER}/$(uname -r)"
+
+rm -f /tmp/o;
+# background scap-open
+if [[ ${COMPILER} == *"clang"* ]]; then
+  /home/vagrant/scap-open --bpf "/home/vagrant/driver/${COMPILER}/$(uname -r).o" > /tmp/o &
```

Comment:
I think we could change this to print the output as it is written to the file instead of doing a `cat` afterwards, WDYT?
```suggestion
  /home/vagrant/scap-open --bpf "/home/vagrant/driver/${COMPILER}/$(uname -r).o" | tee /tmp/o &
```